doc/xml/contributing.xml

```diff
@@ -787,6 +787,7 @@ pgbackrest/doc/doc.pl --out=html --include=user-guide --require=/quickstart --va
         <list>
             <list-item>Provide a short submission title.</list-item>
             <list-item>Write a detailed comment to describe the purpose of your submission and any issue(s), if any, it is resolving; a link to the GitHub issue is also helpful.</list-item>
+            <list-item>Ensure your PR is targeted against the <code>integration</code> branch.</list-item>
```

Comment:
Looking at the github instructions for creating a PR, the merge into/left side is called the base,  so should we try to match the terminology? Something along the lines of
Select the `integration` branch as the base in your PR, do not select `main` nor any other branch.

Reply:
WFM.